BSP/STM32F413/Makefile

```diff
@@ -3,7 +3,9 @@
 ######################################
 TARGET = bps-leader
 
-DEBUG = 0
+# sets debugging mode on or off based on BPS makefile
+DEBUG = $(debug)
```

Comment:
Don't really understand the purpose of this. In the higher level makefile we export "debug", so it is available anywhere to use in this makefile. Why make another variable called "DEBUG" and then set it to "debug"?

imo you just call the one in the higher makefile DEBUG, and then remove this line. Since it's exported it should work the same.

Reply:
u right, I'll change it.